fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -2979,6 +2998,19 @@ public Builder setPipelineSizer(@Nonnull PipelineSizer pipelineSizer) {
             return this;
         }
 
+        @Nullable
+        @Override
```

Comment:
The order of `Nullable` and `Override` should be swapped to match the others in this file.

I don't think we have a consistent order of these things throughout our codebase, but I feel like they should at least match within a file.

Reply:
ok. sure.